docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -38,59 +34,36 @@ spec:
-you can have network intensive {block_storage} volume services with lots of I/O as well as
-{block_storage} backup services that are not only network intensive but also memory and
-CPU intensive. This may be a concern for the {OpenShiftShort} human operators, and
-they may want to use the `nodeSelector` to prevent these service from
-interfering with their other {OpenShiftShort} workloads. For more information about node selection, see xref:about-node-selector_planning[About node selector].
-+
-When selecting the nodes where the {block_storage} volume is going to run remember that {block_storage}-volume may also use local storage when downloading a {image_service_first_ref} image for the create volume from image operation, and it can require a considerable
-amount of space when having concurrent operations and not using {block_storage} volume
-cache.
+For more information about node selection, see xref:about-node-selector_planning[About node selector].
 +
-If you do not have nodes with enough local disk space for the temporary images, you can use a remote NFS location for the images. You had to manually set this up in {OpenStackPreviousInstaller} deployments, but with operators, you can do it
-automatically using the extra volumes feature ()`extraMounts`.
-
+[NOTE]
+====
+If your nodes do not have enough local disk space for temporary images, you can use a remote NFS location by setting the extra volumes feature ()`extraMounts`.
+====
 Transport protocols::
-Due to the specifics of the storage transport protocols some changes may be
-required on the {OpenShiftShort} side, and although this is something that must be
-documented by the Vendor here wer are going to provide some generic
-instructions that can serve as a guide for the different transport protocols.
-+
-Check the backend sections in your `cinder.conf` file that are listed in the
-`enabled_backends` configuration option to figure out the transport storage
-protocol used by the backend.
-+
-Depending on the backend, you can find the transport protocol:
+Due to the specifics of the storage transport protocols, some changes might be
+required for {OpenShiftShort}. Use the following instructions as a guide for the different transport protocols:
 +
-* Looking at the `volume_driver` configuration option, as it may contain the
-protocol itself: RBD, iSCSI, FC...
-* Looking at the `target_protocol` configuration option
+* Check the back-end sections in your `cinder.conf` file that are listed in the
+`enabled_backends` configuration option to determine the transport storage
+protocol that the back end uses.
+* Depending on the back end, you can find the transport protocol by viewing the `volume_driver` or `target_protocol` configuration options.
 +
-WARNING: Any time a `MachineConfig` is used to make changes to {OpenShiftShort}
-nodes the node will reboot!!  Act accordingly.
+[WARNING]
+If you use a `MachineConfig` to make changes to {OpenShiftShort} nodes, the nodes reboot.
```

Comment:
Is the purpose of this note to make customers aware that the node reboots? Or do we NOT want customers' nodes to reboot?

Reply:
It is to let them know that they reboot, because they may be used to the old mechanism in RHEL where starting a service doesn't require a reboot.